app/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesJob.scala

```diff
@@ -36,11 +36,13 @@ class PollingNewMessagesJob @Inject() (
   ernSubmissionRepository: ErnSubmissionRepository,
   messageService: MessageService,
   dateTimeService: DateTimeService
-) extends ScheduledJob {
+) extends ScheduledJob
+    with Logging {
 
   override def name: String = "polling-new-messages-job"
 
   override def execute(implicit ec: ExecutionContext): Future[Done] = {
+    logger.warn("[PollingNewMessagesJob] - Started PollingNewMessagesJob")
```

Comment:
Why prefix with `[PollingNewMessagesJob]` when that will be in the `logger` field in kibana anyway? Also why is this warn?

Reply:
Also we log something similar in the `JobScheduler` so I don't know what the value of this is?